src/api/handlers/action.py

```diff
@@ -230,3 +231,12 @@ def super_admin_list(self, request):
     if err:
       return err
     return MassenergizeResponse(data=actions)
+
+
+  @super_admins_only
+  def search_and_filter_actions(self, request): 
```

Comment:
it doesn't look like search_and_filter is fully implemented.  should it be part of this PR?

Reply:
the search and filter was implemented through the list items route